x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/JobManager.java

```diff
@@ -346,8 +346,8 @@ public void onFailure(String source, Exception e) {
                 }
 
                 @Override
-                public void clusterStatePublished(ClusterChangedEvent clusterChangedEvent) {
```

Comment:
I wonder if ClusterStateUpdateTask should have a final empty method for this? I think it's always the wrong one in that contex.t

Reply:
I like this idea, I've pushed 1b7eeb4 to implement this.